doc/controller-card_poweroff.md

```diff
@@ -0,0 +1,132 @@
+```
+
+After controller-card0 transitions to redundant-role SECONDARY:
+
+```
+/components/component[name=controller0]/state/redundant-role, SECONDARY
+/components/component[name=controller0]/controller-card/config/power-admin-state, POWER_DISABLED
+/components/component[name=controller0]/controller-card/state/power-admin-state, POWER_DISABLED
+/components/component[name=controller0]/state/last-poweroff-reason/trigger, USER_INITIATED
+/components/component[name=controller0]/state/last-poweroff-reason/details, "User initiated Shutdown"
+/components/component[name=controller0]/state/last-poweroff-time, 1706590714681765937
+/components/component[name=controller1]/state/redundant-role, PRIMARY
+/components/component[name=controller1]/controller-card/config/power-admin-state, POWER_ENABLED
+/components/component[name=controller1]/controller-card/state/power-admin-state, POWER_ENABLED
+```
+
+2. A controller-card which is in state/redundant-role=SECONDARY and is config/power-admin-state=POWER_DISABLED should remain powered off, even after a reboot. It is possible that after a reboot, both the controller cards are powered-on. However as soon as the configuration is loaded, the system should power-off the subject controller-card.
+
+3. A CONTROLLER_CARD in redundant-role SECONDARY and state/power-admin-state = POWER_DISABLED cannot transition to redundant-role PRIMARY. If the PRIMARY CONTROLLER_CARD goes down, the device will be offline.
+
+4. When a controller card boots up and loads it’s configuration, power-admin-state may be set to POWER_DISABLED. The CONTROLLER_CARD should then power off and never enter into any controller cardprimary/secondary election process. This also means that an implementation shouldn't start a controller card election process until the configuration is loaded and consumed.
+
+
+## Concerns and possible failure scenarios
+
+
+
+1. If a PRIMARY card malfunctions and ends up in a bootloop, would this approach help?
+
+    **Response:**
+
+
+    In this scenario the expectations are that the implementation takes steps to initiate a controller-card switchover operation. Therefore, the standby controller card takes over the PRIMARY role and the system stabilizes allowing for gRPC connections to be established. In this situation if the operator pushes a gNMI configuration to shutdown the SECONDARY card, the PRIMARY card must be able to power-off the SECONDARY controller-card. In this scenario the implementation (depending on their architecture) may also initiate a shutdown of the faulty card from the new PRIMARY card.
+
+2. Since the operation relies on configuration, it is possible that the failure scenario may kick in before the configuration takes effect post a reboot.
+
+    **Response:**
+
+
+	Response here is the same as "1" above.
+
+
+
+3. Both controller-cards are functional and the the secondary controller card is attempted to be shutdown
```

Comment:
```suggestion
3. Both controller-cards are functional and the the secondary controller card is attempted to be powered off.
```

Here and throughout, use a consistent term such as 'power off' and 'power on' for referring to power-admin-state.  Shutdown might be confused with some other configuration.